src/stores/spaces/SpaceStore.ts

```diff
@@ -428,7 +434,9 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
         if (space === MetaSpace.Home && this.allRoomsInHome) {
             return true;
         }
-
+        if (space === MetaSpace.VideoRooms) {
+            return !!this.matrixClient?.getRoom(roomId)?.isCallRoom();
+        }
```

Comment:
This condition is inconsistent with the one in Algorithm.ts which also checks join rule. What's going on with that?